core/icon.tid

```diff
@@ -1,6 +1,6 @@
 title: $:/core/icon
 tags: $:/tags/Image
 
-<svg id="tw-icon" xmlns="http://www.w3.org/2000/svg" height="22pt" width="22pt" version="1.1" viewBox="0 0 128 128">
```

Comment:
Why have you given it an id? I think it's harmless for the image/svg+xml tiddlers, but it's dangerous here.